device-types/HPE/ProCurve-2810-24G.yaml

```diff
@@ -53,11 +58,19 @@ interfaces:
     type: 1000base-t
   - name: '20'
     type: 1000base-t
-  - name: '21'
+  - name: 21-RJ45
```

Comment:
Are these interface names in the config actually suffixed with `-RJ45`? Can you show us an example?
If not, please revert this change and adjust the SFP ones accordingly.

Reply:
They are not. They are labeled "T" and "S" on the chassis, but there is only one interface in the software that does not have this suffix.
```
2530-8G-PoEP# show int brief

 Status and Counters - Port Status

                   | Intrusion                           MDI  Flow
  Port  Type       | Alert     Enabled Status Mode       Mode Ctrl
  ----- ---------- + --------- ------- ------ ---------- ---- ----
  1     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  2     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  3     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  4     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  5     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  6     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  7     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  8     100/1000T  | No        Yes     Down   1000FDx    MDIX off 
  9     100/1000T  | No        Yes     Down   1000FDx    MDI  off 
  10    1000LX     | No        Yes     Up     1000FDx    NA   off
```

Thats why I used other Aruba switches as a reference and adopted the solution from the Aruba 2530 8G.
I'm not quiet sure how to define this accurately.

Reply:
Yeah they must not be defined correctly then. Can you show me the config for the ports in question here? 21, 22, etc

Reply:
Okay. Sure, here is the output:

```
2810-24G# show int brief

 Status and Counters - Port Status

                  | Intrusion                           MDI   Flow  Bcast 
  Port  Type      | Alert     Enabled Status Mode       Mode  Ctrl  Limit 
  ----- --------- + --------- ------- ------ ---------- ----- ----- ------
  1     100/1000T | No        Yes     Up     1000FDx    MDIX  off   0     
  2     100/1000T | No        Yes     Up     1000FDx    MDI   off   0     
  3     100/1000T | No        Yes     Up     1000FDx    MDI   off   0     
  4     100/1000T | No        Yes     Down   1000FDx    MDIX  off   0     
  ...
  ...
  21    100/1000T | No        Yes     Down   1000FDx    MDI   off   0     
  22    100/1000T | No        Yes     Down   1000FDx    MDI   off   0     
  23    100/1000T | No        Yes     Up     1000FDx    MDIX  off   0     
  24    100/1000T | No        Yes     Down   1000FDx    MDI   off   0
```

Reply:
Yeah so the port names are just `21`, `22`, etc

Reply:
okay, fixed it